syncd/syncd_saiswitch.cpp

```diff
@@ -610,6 +598,58 @@ void SaiSwitch::collectCounters(
     }
 }
 
+void SaiSwitch::collectQueueCounters(
+        _In_ swss::Table &countersTable) const
+{
+    SWSS_LOG_ENTER();
+
+    uint32_t countersSize = (uint32_t)m_supported_queue_counters.size();
+    std::vector<uint64_t> counters;
+    counters.resize(countersSize);
+
+    for (auto &port_rid: m_port_rid_list)
+    {
+        if (m_queue_rid_map.find(port_rid) == m_queue_rid_map.end())
```

Comment:
reuse iterator instead double looking find() and then at90 on line 619, compare iterator with end()